cli/cli/commands/service/inspect/inspect.go

```diff
@@ -143,43 +176,70 @@ func PrintServiceInspect(ctx context.Context, kurtosisBackend backend_interface.
 		userService = userServiceInfo
 		break
 	}
-	out.PrintOutLn(fmt.Sprintf("%s: %s", serviceNameTitleName, userService.GetName()))
+
+	if outputFormat != "" {
+		jsonMap[ServiceImageTitleName] = userService.GetContainer().GetImageName()
+		jsonMap[ServiceCmdArgsTitleName] = userService.GetContainer().GetCmdArgs()
+		jsonMap[ServiceEntrypointArgsTitleName] = userService.GetContainer().GetEntrypointArgs()
+		jsonMap[ServiceEnvVarsTitleName] = userService.GetContainer().GetEnvVars()
+		jsonMap[ServicePortsTitleName] = userService.GetPrivatePorts()
```

Comment:
nit: cleaner would be to populate this map in for all cases and then just have a switch on the outputFormat, where the default ("") would just use out.PrintLn as.

Logic would look like:
1. fetch all fields + populate fields struct
2. switch statement + serialization/output to stdout

Reply:
now that I use the json struct all the fields are fetched and populated into struct and I like the idea of the switch so just added that